packages/frontend/src/renderer/components/widgets/channels/ChannelMessages.tsx

```diff
@@ -93,10 +97,28 @@ export const ChannelMessagesComponent: React.FC<IChannelMessagesProps> = ({
   /* Keep scroll position when new chunk of messages are being loaded */
   useEffect(() => {
     if (scrollbarRef.current && scrollPosition === 0) {
-      scrollbarRef.current.scrollTop = scrollbarRef.current.scrollHeight - scrollHeight
+      setTimeout(() => {
+        scrollbarRef.current.scrollTop = scrollbarRef.current.scrollHeight - scrollHeight
+      })
     }
   }, [messages.count])
 
+  /* Scroll to bottom if new message arrives and scroll is at the top */
+  useEffect(() => {
+    if (
+      scrollbarRef.current &&
+      scrollPosition === 0 &&
+      previousMessages &&
+      // @ts-ignore
```

Comment:
Why we need ts-ignore here?

Reply:
previousMessages and previousSlice have undefined type because on first render they are indeed undefined